### PR TITLE
Fix #2757: Test input/textarea.selection{Direction,Start,End} not throwing

### LINKED ISSUE #2757: Test input/textarea.selection{Direction,Start,End} not throwing
The spec was recently changed in https://github.com/whatwg/html/pull/1006, so some tests probably need updating.

### PR DESCRIPTION
**Pull request by zcorpan**
This tests https://github.com/whatwg/html/pull/1006.

### CODE REVIEW

**Comment by zcorpan** [comment]
@cvrebert r?

File: html/dom/interfaces.html [hunk 1]
```diff
@@ -3351,7 +3358,30 @@
     HTMLFieldsetElement: ['document.createElement("fieldset")'],
     HTMLLegendElement: ['document.createElement("legend")'],
     HTMLLabelElement: ['document.createElement("label")'],
-    HTMLInputElement: ['document.createElement("input")'],
+    HTMLInputElement: [
+      'document.createElement("input")',
```
Comment (cvrebert) [inline html/dom/interfaces.html]: Adding an explicit `text` entry would be good, IMHO.

**Comment by cvrebert** [comment]
Other than that, lgtm.

**Comment by zcorpan** [comment]
Thanks!

**Pull request merged**